examples/lit-icd-app/esp32/main/main.cpp

```diff
@@ -66,6 +81,13 @@ extern const char TAG[] = "lit-icd-app";
 
 static AppDeviceCallbacks EchoCallbacks;
 
+static void UatButtonHandler(UatButton * button)
+{
+    DeviceLayer::PlatformMgr().ScheduleWork([](intptr_t) {
+        Server::GetInstance().GetICDManager().UpdateOperationState(app::ICDManager::OperationalState::ActiveMode);
+    });
+}
```

Comment:
This isn't the right way to interact with the ICDManager. We should instead use the `ICDNotifier` which allows to notify the ICDManager of outside events.

`NotifyNetworkActivityNotification` is the function to use until the UAT one is created.

Reply:
Changed.